internal/provider/provider.go

```diff
@@ -97,11 +120,16 @@ func (p *tfmProvider) Configure(ctx context.Context, req provider.ConfigureReque
 	// with Terraform configuration value if set.
 
 	githubToken := os.Getenv(GITHUB_TOKEN_ENV_NAME)
+	hostname := HCP_TERRAFORM_HOST
 
 	if !config.GithubToken.IsNull() {
 		githubToken = config.GithubToken.ValueString()
 	}
 
+	if !config.Hostname.IsNull() {
+		hostname = config.GithubToken.ValueString()
```

Comment:
Is this correct, we are assigning `GithubToken` to `hostname` field, seems a little ambiguous, if it is correct please add a comment to clarify the context to future devs

Reply:
ohh shoot, this is a miss dude.. Thanks for catching it.